### PR TITLE
feat(app): improved socket.io logging

### PR DESCRIPTION
**Pull request by kingcody**

### CODE REVIEW

File: app/templates/server/config/socketio(socketio).js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function onDisconnect(socket) {
 function onConnect(socket) {
   // When the client emits 'info', this listens and executes
   socket.on('info', function(data) {
-    console.info('[%s] %s', socket.address, JSON.stringify(data, null, 2));
+    console.info(`SocketIO ${socket.nsp.name} [${socket.address}] ${JSON.stringify(data, null, 2)}`);
```
Comment (kingcody) [inline app/templates/server/config/socketio(socketio).js]: This line needs to be `<=100`.

**Comment by kingcody** [comment]
@Awk34 what do you think about this change? (sans jscs failure)

File: app/templates/server/config/socketio(socketio).js (hunk 1 above)
Comment (Awk34) [inline app/templates/server/config/socketio(socketio).js]: characters long?

**Comment by Awk34** [comment]
LGTM

**Comment by kingcody** [comment]
Sounds good, I'll fix the line length.

**Comment by kingcody** [comment]
@Awk34 check it again, I added a `socket.log` method.

**Pull request merged**